doc-Provisioning_Virtual_Machines_and_Hosts/topics/Catalogs_and_Services.adoc

```diff
@@ -364,4 +364,165 @@ include::common/catalog-order.adoc[]
 The parameters are passed to the children based on the method tied to the choices made in the dialog.
 
 
+[[example-service-provisioning]]
+=== Example: Creating a Service
+
+{product-title} provides the ability to define services, which can include one or more virtual machines or instances, and can be deployed across hybrid environments. Services can be made available to users through the self-service user interface (SUI) that allows users to order predefined IT services without IT operations getting involved.
+
+==== Creating a Service for Provisioning a Virtual Machine through Red Hat Virtualization (RHV)
+
+
+The following three items are required to make a service available to users from the {product-title_short} self-service user interface:
+
+. *Service Dialog*: A service dialog is where you allow users to configure virtual machine or instance options. A service dialog determines which options users get to change. You can choose the options presented to the user. You could just give them the option to set the service name, or you could have them change all of the provisioning dialog options. 
+
+. *Service Catalog*: A service catalog is used to group service dialogs together.
+
+. *Service Catalog Item*: A service catalog item is the actual service which combines a service dialog with a provisioning dialog.
+
+
+.Examining the Provisioning Dialog
+
+To work with services in CloudForms it is important to understand the concept of provisioning dialogs. Every provider in {product-title_short} comes with a sample provisioning dialog covering the options specific to that provider. A provisioning dialog presents the basic configuration options for a virtual machine or instance. When you begin the process of provisioning a virtual machine or instance via {product-title_short}, you are presented with a provisioning dialog where you set certain options for the virtual machine or instance. The options presented are dependent on the provider you are using. For instance, a cloud provider might have flavors of instances, whereas an infrastructure provider might allow you to set the memory size or number of CPUs on a virtual machine.
+
+To have a look at the sample provisioning dialog for Red Hat Virtualization, go to menu:Automation[Automate > Customization > Provisioning Dialogs > VM Provision] and select *Sample RedHat VM Provisioning Dialog*. This is a textual representation of the dialog you will get when you provision a virtual machine or instance.
```

Comment:
Do you think here the path can be broken down, as it follows in the GUI to inform the user that they need to select a folder from an accordian menu tab? This could actually make a decent case for a doc convention.

Reply:
Good point, fixed. Sure, let's bring this up in our next team meeting.